tests/E2E/package.json

```diff
@@ -35,7 +35,7 @@
     "parsed-url": "0.0.0",
     "q": "^1.4",
     "req": "^0.1",
-    "selenium-standalone": "^6.14.0",
+    "selenium-standalone": "^6.16",
```

Comment:
why do we need selenium if we remove the selenium test suite ?

Reply:
Selenium is still needed to run our tests under `tests/E2E`. We only remove `tests/Selenium`